docs/src/cargo-toml-configuration.md

```diff
@@ -0,0 +1,29 @@
+# `Cargo.toml` Configuration
+
+`wasm-pack` can be configured via the `metadata.wasm-pack` key in
+`Cargo.toml`. Every option has a default, and is not required.
+
+There are three profiles: `dev`, `profiling`, and `release`. These correspond to
+the `--dev`, `--profiling`, and `--release` flags passed to `wasm-pack build`.
+
+The available configuration options and their default values are shown below:
+
+```toml
+[metadata.wasm-pack.profile.dev.wasm-bindgen]
+# Should we enable wasm-bindgen's debug assertions in its generated JS glue?
+debug = true
+# Should wasm-bindgen demangle the symbols in the "names" custom section?
+demangle = true
+# Should we keep the DWARF debug info custom sections?
+keep-debug = false
```

Comment:
I'm not a huge fan of the name added to `wasm-bindgen` here, maybe we could be a bit more descriptive here with something like:

```toml
# Whether or not the DWARF debug info custom sections are preserved
dwarf-debug = true
```

Is the default `false` here because they're somewhat non-operational today?

Reply:
> Is the default `false` here because they're somewhat non-operational today?

It was true for `--dev` builds in the last iteration of the PR and you requested it be turned off because nothing was consuming it yet :-p

Reply:
Oh lol I can't seem to make up my mind...

Sounds like I'm fine with whatever, although we may wish to have a small comment in the documentation indicating that nothing known consumes this and the info itself is probably wrong given what wasm-bindgen does to the binary.

I think I actually like the idea of leaving it off-by-default